specifyweb/frontend/js_src/lib/components/FormMeta/CarryForward.tsx

```diff
@@ -399,7 +399,9 @@ function CarryForwardCategory({
           const isUnique = uniqueFields.includes(field.name);
           const isRequired =
             isBulkConfig === true &&
-            (field.localization.isrequired || field.overrides.isRequired);
+            (field.localization.isrequired ||
+              field.overrides.isRequired ||
+              field.databaseColumn === 'CollectionObjectTypeID');
```

Comment:
hmm, are you sure that the bug in #5207 is affecting only this one field among all the fields in the database?


also, the condition here is only affecting the UI of this dialog - I don't think it has impact on the actual carry over operation, does it?

Reply:
yes it does; the COType is now being carried away when bulk carrying.
The issue here is that the COType wasn't required (so we could uncheck it) but we have now a rule that says that you always need a cotype and that this needs to be from the same treedef as the determination. Which was throwing the errors. 
Might have to do something similar in the next release for COGType